modules/ROOT/pages/secure-storage.adoc

```diff
@@ -0,0 +1,110 @@
+= Secure Storage Overview
```

Comment:
Also, this is not really an overview, because it has several very complete subtopics, including two tasks.
Did you intend for the tasks to have H1 titles?
Otherwise, this page should be retitled to imply audience participation, not just brief overview points.